client/allocdir/alloc_dir_windows.go

```diff
@@ -3,6 +3,15 @@ package allocdir
 import (
 	"errors"
 	"os"
+	"path/filepath"
+)
+
+var (
+	//Path inside container for mounted directory that is shared across tasks in a task group.
+	SharedAllocContainerPath = filepath.Join("c:\\", SharedAllocName)
```

Comment:
Well, I am not sure how bind mount sharing between containers and hosts work in windows. Can you please explain how the file system is laid out inside a windows container? So if it's always going to be `C:\\` I think this is fine.